lisa/schema.py

```diff
@@ -735,12 +735,15 @@ class Criteria:
     )
 
 
+@dataclass_json(undefined=Undefined.EXCLUDE)
+@dataclass
 class BaseTestCaseFilter(TypedSchema, BaseClassMixin):
     """
     base test case filters for subclass factory
     """
 
-    ...
+    # if it's false, the test cases are disable in current run.
```

Comment:
This sentense is confusing. If it's false, this filter is not effective. test cases can be affected by anotther filter.